test/thermodynamics/change_in_entropy_of_ideal_gas_from_volume_and_temperature_test.py

```diff
@@ -0,0 +1,85 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    assert_equal
+)
+
+from symplyphysics.laws.thermodynamics import change_in_entropy_of_ideal_gas_from_volume_and_temperature as change_in_entropy
+
+## Source of numbers: https://edu.tltsu.ru/er/book_view.php?book_id=48e&page_id=3918
+
+@fixture(name="test_args")
+def test_args_fixture():
+    mass = Quantity(0.008 * units.kilograms)
+    molar_mass = Quantity(0.032 * units.kilograms / units.mole)
+    molar_heat_capacity = Quantity(29.4 * units.joule / (units.kelvin * units.mole))
+    final_temperature = Quantity(573 * units.kelvin)
+    initial_temperature = Quantity(353 * units.kelvin)
+    final_volume = Quantity(0.04 * units.meter**3)
+    start_volume = Quantity(0.01 * units.meter**3)
+    Args = namedtuple("Args", ["mass", "molar_mass", "molar_heat_capacity", "final_temperature", "initial_temperature", "final_volume", "start_volume"])
+    return Args(
+        mass=mass,
+        molar_mass=molar_mass,
+        molar_heat_capacity=molar_heat_capacity,
+        final_temperature=final_temperature,
+        initial_temperature=initial_temperature,
+        final_volume=final_volume,
+        start_volume=start_volume
+    )
+
+
+def test_basic_law(test_args):
+    result = change_in_entropy.calculate_entropy_change(test_args.mass, test_args.molar_mass, test_args.molar_heat_capacity, test_args.final_temperature, test_args.initial_temperature, test_args.final_volume, test_args.start_volume)
+    assert_equal(result, 6.442 * (units.joule / units.kelvin))
```

Comment:
Did you use example #7 as reference? I see it results to 5.4 of something, probably J/K.